optin/reporting/controllers/optinRequestsController.js

```diff
@@ -57,15 +54,15 @@ const controller = {
 			}
 		};
 
-		const status = statusStringToInt(statusFilter.toLowerCase());
+		const statusFilter = statusStringToInt(status.toLowerCase());
```

Comment:
why did you rename this? if it is single value (int), i think status name is clearer. "filter" typically connotates a function.

Reply:
I rename that because I did destructure params. I will undo this part.